### PR TITLE
ZANGCPAAS-4022: Updating base URLs for different deployments

### PR DESCRIPTION
**Pull request by ajitha-avaya**
JIRA:
https://jira.forge.avaya.com/browse/ZANGCPAAS-4022

Required for hackathon event

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -36,6 +36,12 @@ In order to authorize against Avaya OneCloud™️ CPaaS  services you'll have t
     
 .. code-block:: bash
 
+You can also change the base API url in /constants.go. The default is set to "https://api.zang.io".
```
Comment (prasadrotti) [inline README.rst]: Looks `/v2` is missing.

Comment (ajitha-avaya) [reply README.rst]: Adhering to the value as mentioned here: https://github.com/zang-cloud/zang-go/blob/master/constants.go

Comment (prasadrotti) [reply README.rst]: ok, but for new US and EU, are we using `/v2`

Comment (ajitha-avaya) [reply README.rst]: yes we do.

**Review by prasadrotti — APPROVED** [review]

**Review by ssomen — APPROVED** [review]

File: README.rst [hunk 2]
```diff
@@ -36,6 +36,12 @@ In order to authorize against Avaya OneCloud™️ CPaaS  services you'll have t
...
+
+The base API URL for US(new) and EU deployments are:
+US: https://api-us.cpaas.avayacloud.com
+EU: https://api-eu.cpaas.avayacloud.com
```
Comment (ssomen) [inline README.rst]: Why there is no '/v2' here ? but present in https://github.com/zang-cloud/zang-dotnet/pull/13/files

Comment (ssomen) [reply README.rst]: Just make sure that '/v2' added internally by SDK. If yes. this is fine

**Comment by ajitha-avaya** [comment]
Hi @ssomen and @prasadrotti : added a example as suggested by @srinivasbanda519 . and added /v2 as well. so that it wont confuse hackathon developers

**Review by prasadrotti — APPROVED** [review]

**Review by ssomen — APPROVED** [review]

**Review by srinivasbanda519 — APPROVED** [review]

**Pull request merged**